.mockery.yaml

```
@@ -0,0 +1,149 @@
+with-expecter: true
+issue-845-fix: True
+resolve-type-alias: False
+dir: "{{ .InterfaceDir }}/../mocks"
+outpkg: "mocks"
+filename: "{{ .InterfaceName | snakecase | lower }}.go"
```

Let's have this pattern as default and then we can remove the override of this config in the specific packages. WDYT?
```suggestion
filename: "mock_{{ .InterfaceName | snakecase | lower }}.go"
```

---

i think we can handle this where we will be standardizing everything